packages/obojobo-document-engine/src/scripts/oboeditor/stores/editor-store.js

```diff
@@ -0,0 +1,229 @@
+/* eslint eqeqeq: 0 */
```

Comment:
In the future we'll need to combine this code and the editor nav code. This seems like it'll cause errors if we update one and not the other.

I know we're fast tracking this and don't have a good way to have both repos include code yet, so it's okay for now.